backend/app/services/email/email_service_provider.py

```diff
@@ -1,12 +1,72 @@
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+from app.interfaces.email_service import EmailContent, EmailTemplate
 from app.interfaces.email_service_provider import IEmailServiceProvider
 
 
 class AmazonSESEmailProvider(IEmailServiceProvider):
-    def __init__(self, aws_access_key: str, aws_secret_key: str):
-        pass
-
-    # TODO (Mayank, Nov 30th) - Create an email object to pass into this method
-    def send_email(
-        self, recipient: str, subject: str, body_html: str, body_text: str
-    ) -> dict:
-        pass
+    def __init__(
+        self,
+        aws_access_key: str,
+        aws_secret_key: str,
+        region: str,
+        source_email: str,
+        is_sandbox: bool = True,
```

Comment:
what does `is_sandbox` do? For quick glancers, would you be able to leave a comment describing this option.